### DISCUSSION TITLE
REQ: Add information on the console on startup

### DISCUSSION THREAD

**Discussion by mushis** [ezQuake/ezquake-source#514] [Ideas]
when launching ezQuake there could be more information on the console about recommended settings for running the game. for example:
- if Threaded Optimization is ON, suggest user to disable it on the gfx card settings
- If the user is running a non-nvidia card, suggest some values for cvars to improve performance
- (more)..

**Comment by jite** [comment]
Thanks. Please fill in (more) with what you miss and find is useful information. Oh and btw, take a look at the Wiki page about posting issues :)
